frontend/src/FloodsRoutes.js

```diff
@@ -5,23 +5,19 @@ import PrivateRoute from './PrivateRoute';
 import Header from './Dashboard/Header/Header';
 import ManageUsers from './Dashboard/ManageUsersPage/ManageUsers';
 import CreateUser from './CreateUser';
-import AdminCrossingList from './AdminCrossingList';
 import CrossingUpdates from './Dashboard/CrossingUpdatesPage/CrossingUpdates'
 import CrossingMap from './Map/CrossingMap';
 import NewStatusUpdate from './NewStatusUpdate';
 import auth from './services/gqlAuth';
 import { graphql } from 'react-apollo';
 import gql from 'graphql-tag';
 
-const Protected = () => <h3>Protected</h3>;
-
-class FloodsAdminRoutes extends Component {
+class FloodsRoutes extends Component {
   render() {
     return (
         <div>
           <Route path="/dashboard" render={(props) => <Header currentUser={this.props.data.currentUser} {...props} />} />
-          <Route path="/crossings" component={AdminCrossingList}/>
-          <Route path="/dashboard/map" component={CrossingMap}/>
+          <Route path="/dashboard/map" component={CrossingMap} currentUser={this.props.data.currentUser}/>
```

Comment:
just making a note that the way we pass the `currentUser` props in line 20 is inconsistent from line 19. Maybe this is by necessity since the `/dashboard` route needs more props passed through? My worthless opinion is that I think the `component={Component}` is preferable over `render={() => <Component />}` because its easier to understand.